test/browser/spec/ol/format/ows.test.js

```diff
@@ -79,7 +79,7 @@ describe('ol.format.OWS 1.1', function () {
     const serviceIdentification = obj.ServiceIdentification;
     expect(serviceIdentification).to.be.ok();
     expect(serviceIdentification.Abstract).to.eql(
-      'Service that contrains the map access interface to some TileMatrixSets'
+      'Service that contains the map access interface to some TileMatrixSets'
```

Comment:
Looks like this also needs to be corrected in these places:
 * https://github.com/openlayers/openlayers/blob/0519e908d55fbeb95408a06644930adfe2256cda/test/browser/spec/ol/format/wmts/ogcsample.xml#L5
 * https://github.com/openlayers/openlayers/blob/0519e908d55fbeb95408a06644930adfe2256cda/test/browser/spec/ol/format/ows.test.js#L62

Reply:
Should it be `contains` or `constrains` ?

Reply:
Should indeed be `constrains`.

Reply:
@luzpaz - in case it helps, it doesn't really matter what the content is, it just needs to be the same where we pass it to the [`parse` function](https://github.com/openlayers/openlayers/blob/b03ba31affe77dfb1bf190118a5454455504a043/test/browser/spec/ol/format/ows.test.js#L62) and where we pass it to the [`eql` function](https://github.com/openlayers/openlayers/blob/b03ba31affe77dfb1bf190118a5454455504a043/test/browser/spec/ol/format/ows.test.js#L82).

(The `<ows:AccessConstraints>` element is not really related to the content of the `<ows:Abstract>`, but the original author probably did mean to write "Service that constrains the map access interface..." for the abstract.)

Reply:
Understood. Here are the requested revisions: 1bc0f6a